src/mds/MDSRank.cc

```diff
@@ -576,9 +576,9 @@ bool MDSRank::handle_deferrable_message(Message *m)
       // SERVER
     case CEPH_MSG_CLIENT_SESSION:
     case CEPH_MSG_CLIENT_RECONNECT:
-      ALLOW_MESSAGES_FROM(CEPH_ENTITY_TYPE_CLIENT);
       // fall-thru
     case CEPH_MSG_CLIENT_REQUEST:
+      ALLOW_MESSAGES_FROM(CEPH_ENTITY_TYPE_CLIENT);
```

Comment:
Hmm, in MDSRank::forward_message_mds we convert client-sent MSG_CLIENT_REQUESTs into MClientRequestForwards back to the client, rather than directly sending on to another MDS.

I can't see a path where we actually send an MClientRequest from one mds to another, but it's not the easiest thing to grep for